tests/util/helm_utils.go

```diff
@@ -33,15 +33,25 @@ func HelmDepUpdate(chartDir string) error {
 }
 
 // HelmInstallDryRun helm install dry run from a chart for a given namespace
-func HelmInstallDryRun(chartDir, chartName, namespace, setValue string) error {
-	_, err := Shell("helm install --dry-run --debug %s --name %s --namespace %s %s", chartDir, chartName, namespace, setValue)
+func HelmInstallDryRun(chartDir, chartName, namespace, valueFile, setValue string) error {
+	var err error
+	if valueFile != "" {
+		_, err = Shell("helm install --dry-run --debug %s --name %s --values %s --namespace %s %s", chartDir, chartName, valueFile, namespace, setValue)
```

Comment:
may want to construct this dynamically? adding more options later will multiply the combinations.